README.md

```diff
@@ -17,9 +17,50 @@ The path to dynawo deploy is then the path to the subdirectory `dynawo` in the d
 PATH_TO_DYNAWO_DEPLOY=${DYNAWO_HOME}/deploy/gcc8/shared/dynawo/
 ```
 
-Then generate the cmake cache in your build directory:
+### By script
 
-`cmake <SRC_DIR> -DCMAKE_BUILD_TYPE=Release -DDYNAWO_HOME=<DYNAWO_HOME> -DBOOST_ROOT=<DYNAWO_HOME> -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>`
+To build DynaFlow Launcher you need to clone the repository and launch the following commands in the source code directory, it will create a `myEnvDFL.sh` file that will be your personal entrypoint to launch DFL and parametrise some options.
+
+``` bash
+$> git clone https://github.com/dynawo/dynaflow-launcher
+$> cd dynaflow-launcher
+$> echo '#!/bin/bash
+
+# Required
+export DYNAFLOW_LAUNCHER_HOME=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+export DYNAWO_HOME=<PATH_TO_DYNAWO_DEPLOY>
+export DYNAFLOW_LAUNCHER_BUILD_TYPE=Debug
+export DYNAFLOW_LAUNCHER_LOCALE=en_GB
+
+# Optional
+# export DYNAFLOW_LAUNCHER_CMAKE_GENERATOR=Ninja # default is Unix Makefiles
+# export DYNAFLOW_LAUNCHER_PROCESSORS_USED=8 # default 1
+# export DYNAFLOW_LAUNCHER_SHARED_LIB=ON # default OFF
+# export DYNAFLOW_LAUNCHER_USE_DOXYGEN=OFF # default ON
+# export DYNAFLOW_LAUNCHER_BUILD_TESTS=OFF # default ON
+# export DYNAFLOW_LAUNCHER_LOG_LEVEL=INFO # default INFO: can be DEBUG, INFO, WARN, ERROR
```

Comment:
The log level is decided at the compilation ? If so it would not be used friendly (as the user will use a release and not compile by himself) so we would need to open a ticket on this topic

Reply:
default log level is decided at compile time. log level is decided at runtime